src/inverseRobot.js

```diff
@@ -7,7 +7,46 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const outRobot = { ...robot };
+  const newRobot = {};
+  const keysArr = [];
+  const valuesArr = [];
+
+  for (const key in outRobot) {
+    keysArr.push(key);
+  }
```

Comment:
You don't need this, try to rewrite it. But for the future, if you want to get an array of all keys you can use [Object.keys()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)